package.json

```diff
@@ -32,6 +32,7 @@
     "preview": false,
     "qna": "https://github.com/aws/aws-toolkit-vscode/issues",
     "activationEvents": [
+        "onStartupFinished",
```

Comment:
What will happen if you don't bump the vs code minver to 1.46 (https://code.visualstudio.com/updates/v1_46#_onstartupfinished-activation-event)

Reply:
Tested on 1.42. It's silently ignored.

Interestingly it works on 1.44 as well so looks like it just wasn't advertised until 1.46